docs/source/getting_started/estimation/first_msm_estimation_with_estimagic.ipynb

```diff
@@ -7,24 +7,24 @@
    "source": [
```

Comment:
<p>", there can be other ways to calculate the covariance matrix" (not moments).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/estimagic/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>